modules/olm-determining-catalog-compatibility-with-restricted-psa.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * operators/admin/olm-managing-custom-catalogs.adoc
+
+:_content-type: PROCEDURE
+[id="olm-determining-catalog-compatibility-with-restricted-psa_{context}"]
+= Determining catalog source pod compatibility with restricted pod security enforcement
```

Comment:
this procedure only works to diagnose a catalog pod that is already failing because you're trying to run it in a restricted environment.

it does not help you determine ahead of time, whether the catalog will be able to run in a restricted environment.

While the former is useful, the latter would be more useful.  (Also this documentation doesn't make it clear that this is the case, so someone might do this check on a 4.12 cluster and assume they are ok, when they are not)

Reply:
Based on a conversation in Slack with @joelanford, I am going to pull this into a separate PR so we can iterate on this procedure without holding up the rest of the documentation.